bundles/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/values/PercentageValue.java

```diff
@@ -47,10 +48,11 @@
  */
 @NonNullByDefault
 public class PercentageValue extends Value {
-    private final double min;
-    private final double max;
-    private final double span;
-    private final double step;
+    private static final BigDecimal DB100 = BigDecimal.valueOf(100);
```

Comment:
I think `HUNDRED` would make more sense and/or you turned the letters of BigDecimal around